ndcube/utils/cube.py

```diff
@@ -197,3 +199,101 @@ def get_crop_item_from_points(points, wcs, crop_by_values):
         raise ValueError("Input points causes cube to be cropped to a single pixel. "
                          "This is not supported.")
     return tuple(item)
+
+
+def propagate_rebin_uncertainties(uncertainty, data, mask, **kwargs):
```

Comment:
Because it's specific to the API for propagating uncertainties within the realm of `NDCube.rebin`.  If/when `rebin` is upstreamed, this function should go with it.